readme.md

```diff
@@ -1,21 +1,28 @@
 # Frontend practice with catalog page
+
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_catalog/report/html_report/)
+
+- [DEMO LINK](https://Oleksandr-Hundertailo.github.io/layout_catalog/)
+- [TEST REPORT LINK](https://Oleksandr-Hundertailo.github.io/layout_catalog/report/html_report/)
```

Comment:
We need to add these links to the description of the pull request

Reply:
Add links to description